src/main/rx/ghst_protocol.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#define GHST_RX_BAUDRATE                420000
+
+#define GHST_TX_BAUDRATE_FAST		    400000
+#define GHST_TX_BAUDRATE_SLOW	    	115200
+#define GHST_BYTE_TIME_US				25				// (1/GHST_BAUDRATE) * 10000000
```

Comment:
If this is a calculated value then it should rather be calculated, and the comment avoided.

Reply:
Done